### DISCUSSION TITLE
WCAG 2.3.1 Three Flashes or Below Threshold

### DISCUSSION THREAD

**Discussion by evansque** [w3c/low-vision-a11y-tf#154] [Q&A]
I am working on a flash detection tool based on WCAG SC 2.3.1
from [WCAG 2.3.1 SC](https://www.w3.org/WAI/WCAG21/Understanding/three-flashes-or-below-threshold.html)

> A general flash is defined as a pair of opposing changes in [relative luminance](https://www.w3.org/WAI/WCAG21/Understanding/three-flashes-or-below-threshold.html#dfn-relative-luminance) of 10% or more of the maximum relative luminance where the relative luminance of the darker image is below 0.80; and where "a pair of opposing changes" is an increase followed by a decrease, or a decrease followed by an increase

Do the increase/decrease values have to be sequential in a one second period?

Here is an example:
In a 25f/s video (f1, f2, f3, f4, f5, f6 ... f25). does the following changes in luminance values count as a flash?
f1-f2 4% increase
f2-f3 4% increase
f3-f4 4% increase
f4-f5 4% decrease

or does it have to be 10% or more from one frame to another?

**Comment by mraccess77** [comment]
Hi, I would recommend you put this issue on the [WCAG github](https://github.com/w3c/wcag) repository as there would be more resources there to answer it.
